.env.example

```diff
@@ -7,3 +7,6 @@ LOG_LEVEL=debug
 
 # Go to https://smee.io/new set this to the URL that you are redirected to.
 WEBHOOK_PROXY_URL=
+SLACK_CRON_PATTERN=* * * * *
```

Comment:
We should provide these new environment variable in k8s deployment, through `ConfigMap` or `Secret`.

Reply:
yep, was waiting for the k8s PR to get merged before doing this